app/models/contributor.rb

```diff
@@ -69,11 +69,9 @@ def self.with_lowercased_email(email)
   end
 
   def reply(message_decorator)
-    message = message_decorator.message
```

Comment:
Won't this change have a big impact on other messengers besides Whatsapp? I think the method `reply` is called for other messengers but this PR doesn't touch them, right?

Reply:
This change was an intermittent change that was reverted to favor setting the request in the adapter itself.
If you run a git diff between the HEAD of this PR and the main branch for the contributor model, you will see that there are no changes.